releases/Process/Major Release Process.md

```diff
@@ -1,18 +1,16 @@
-> Written with [StackEdit](https://stackedit.io/).
-> <![endif]-->
 
 # Draft - O3DE Major Release Process
-V1.0 Updated December 5, 2022. 
-The release process is managed by sig-release. To request an update to this document, please open an issue at https://github.com/o3de/sig-release/issues
+V1.1 Updated December 22, 2022. 
+The release process is managed by the Release Special Interest Group (SIG). To request an update to this document, please open an issue at https://github.com/o3de/sig-release/issues
 
 ## Key Details
 *Intent: Outlines the high-level information about O3DE releases.*
 
-* Releases are owned and managed by sig-release.
-* Sig-release relies on the help of others in the O3DE community (outside of SIG-Release). See "Roles and Responsibilities" below
-* O3DE Major Releases occur twice each year: approximately in April and October. These dates are discussed in SIG-Release meetings.
-* O3DE Minor Releases occur as needed, as decided by sig-release.
-* Releases are named  according to recommendations from the marketing committee as outlined at https://github.com/o3de/sig-release/issues/39
+* Releases are owned and managed by the Release SIG.
+* The Release SIG relies on the help of others in the O3DE community (outside of SIG-Release). See "Roles and Responsibilities" below
+* O3DE Major Releases occur twice each year: approximately in April and October. These dates are discussed in Release SIG meetings.
+* O3DE Minor Releases occur as needed, as decided by the Release SIG.
+* Releases are named  according to recommendations from the O3DE marketing committee as outlined at https://github.com/o3de/sig-release/issues/39
 * Each release managed using a a Github project board with the set of release process tasks.
 * Releases will update the following Repositories:
```

Comment:
If O3DEs extra is included as part of the release, is there any impact on the process?

Reply:
The process doesn't change - it's just one more repository to monitor. During the release process the release manager will need to monitor all of the repositories involved in the release, any bugs being created in those repositories, and PRs. 
Any repository that is included in an O3DE Release can block a release if the repository is not declared stable.

Reply:
Sounds good. Thanks!